test/e2e-load/text/mutable-text.js

```diff
@@ -100,19 +100,17 @@ module.exports = (text) => {
     //   }
     // }
 
-
     // let expectedLength = initialText.length + Array.from(replicas.values()).reduce((acc, result) => {
     //   return result.ops.reduce((op) => op[0] === '-' ? -1 : 1, 0)
     // }, 0)
     // if (expectedLength !== size) {
     //   return reject(new Error(`after processing ops, final text length is not the expected. Expected ${expectedLength} and have ${size}`))
     // }
-
   }
 
   function validate () {
     if (!validation) {
-      validation = new Promise((_resolve, _reject) => {
+      validation = new Promise((resolve, reject) => {
```

Comment:
what's the purpose of this change? you end up setting the module-level resolve/reject imports to undefined

I think the linter may have led you astray

Reply:
I've reverted this change.

Reply:
Yup, it was automatically fixed by the linter, then missed by my when committing. Sorry about that.